src/main/java/org/teamresistance/frc/UltraSensor.java

```diff
@@ -0,0 +1,35 @@
+package org.teamresistance.frc;
+
+import org.strongback.command.Command;
+
+import edu.wpi.first.wpilibj.Ultrasonic;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ * @author Tarik C. Brown
+ */
+public class UltraSensor extends Command {
+  public Ultrasonic ping1;
+  public Ultrasonic ping2;
+  public double range;
+  public double range2;
+  public UltraSensor() {
+    // Should be Ultrasonic(Ping Channel, Echo Channel
+    // Sets the channels and makes the sensors start running in auto
+    ping1 = new Ultrasonic(0,1);
+    ping2 = new Ultrasonic(2,3);
+    ping1.setAutomaticMode(true);
+    ping2.setAutomaticMode(true);
+  }
+  @Override
+  public boolean execute() {
+    //gets the distance from ping sensors
+     range = ping1.getRangeInches();
+     range2 = ping2.getRangeInches();
+    // I believe this will display the distance on the smart dashboard
```

Comment:
I believe you're right, yup. With Strongback's ultrasonic, you'll want to update these lines to:

```java
SmartDashboard.putNumber("Ping distance", yourPingSensor.getDistanceInInches());
```